packages/aggregator/src/__tests__/createAggregator.test.js

```diff
@@ -1,9 +1,9 @@
-import { expect } from 'chai';
-
 import { DataBroker } from '@tupaia/data-broker';
 import { Aggregator } from '../Aggregator';
 import { createAggregator } from '../createAggregator';
 
+jest.mock('@tupaia/data-broker');
```

Comment:
This fixed the database related issues... `DataBroker[14]` calls `getModels()`, which creates a database instance. We don't need the actual database, or even `DataBroker` here so it's safe (and actually a good practice) to mock it.

Those issues should exist in `mocha` too, but for some reason it seems to be more forgiving with open handles than `jest`.

Reply:
Thanks for the clear explaination. I've learned how you mock modules with the best strategy!